src/converters/hvac_converter.py

```
@@ -0,0 +1,94 @@
+        """
+        Processes the HVAC block from YAML, validating and adding each component.
+        """
+        self.logger.info("HVAC Converter Starting...")
+        hvac_data = data.get("HVAC", {})
+        if not hvac_data:
+            self.logger.info("No HVAC data found in YAML.")
+            return
+
+        try:
+            validated_hvac_schema = self.validate(hvac_data)
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)